src/Core/Types/Types/Descriptors/ArgumentDescriptorBase~1.cs

```diff
@@ -45,6 +45,20 @@ public void Type<TInputType>()
                 TypeContext.Input);
         }
 
+        public void Type(Type type)
```

Comment:
The above method should now lean on this non-generic method.

Reply:
added